File: py/IEC61968/InfIEC61968/InfERPSupport/ErpRecLineItem.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
from IEC61968.Common.Status import Status
from IEC61968.InfIEC61968.InfERPSupport.ErpIdentifiedObject import ErpIdentifiedObject
from IEC61968.InfIEC61968.InfERPSupport.ErpJournalEntry import ErpJournalEntry
from IEC61968.InfIEC61968.InfERPSupport.ErpPayment import ErpPayment
from IEC61968.InfIEC61968.InfERPSupport.ErpReceivable import ErpReceivable


class ErpRecLineItem(ErpIdentifiedObject):

    def __init__(self):
        super().__init__()
        self.status = Status()
        self.erp_journal_entries = ErpJournalEntry()
        self.erp_receivable = ErpReceivable()
        self.erp_payments = ErpPayment()

File: java/IEC61968/InfIEC61968/InfERPSupport/ErpRecLineItem.java
import IEC61968.Common.Status;

/**
 * Individual entry of an ErpReceivable, it is a particular transaction
 * representing an invoice, credit memo or debit memo to a customer.
 * @created 25-Dec-2023 8:45:21 PM
 */
public class ErpRecLineItem extends ErpIdentifiedObject {

	public Status status;
	public ErpJournalEntry ErpJournalEntries;
	public ErpReceivable ErpReceivable;
	public ErpPayment ErpPayments;

	public ErpRecLineItem(){

	}
}//end ErpRecLineItem